operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -61,36 +61,29 @@ local filename = 'mimir-writes.json';
       .addPanel(
         local title = 'In-memory series';
         $.panel(title) +
-        $.statPanel(|||
-          sum(cortex_ingester_memory_series{%(ingester)s}
-          / on(%(group_by_cluster)s) group_left
-          max by (%(group_by_cluster)s) (cortex_distributor_replication_factor{%(distributor)s}))
-        ||| % ($._config) {
-          ingester: $.jobMatcher($._config.job_names.ingester),
-          distributor: $.jobMatcher($._config.job_names.distributor),
-        }, format='short') +
+        $.statPanel(
+          $.queries.ingester.ingestOrClassicDeduplicatedQuery('cortex_ingester_memory_series{%s}' % [$.jobMatcher($._config.job_names.ingester)]),
+          format='short'
+        ) +
         $.panelDescription(
           title,
           |||
             The number of series not yet flushed to object storage that are held in ingester memory.
+            Series are deduplicated according to the replication strategy.
```

Comment:
What is this trying to say? In ingest-storage, there's no replication strategy. (Comment below for ingestion rate "With ingest storage we take the maximum rate of each ingest partition." is more understandable to me)

Reply:
👍 updated in https://github.com/grafana/mimir/pull/8175/commits/244067fc518f6b142873b934e447c71dcb791385